docs/source/conf.py

```diff
@@ -75,7 +75,7 @@
 #
 # This is also used if you do content translation via gettext catalogs.
 # Usually you set "language" from the command line for these cases.
-language = None
+language = 'en'
```

Comment:
This is unrelated to the rest of the PR, but I took advantage of this PR to quickly clean up `conf.py`.